src/classes/STG_SettingsManager_TEST.cls

```diff
@@ -1827,8 +1847,14 @@ private with sharing class STG_SettingsManager_TEST {
     @isTest
     private static void testPanelHealthCheck() {
         STG_PanelHealthCheck_CTRL panel = new STG_PanelHealthCheck_CTRL();
-        system.assertEquals('idPanelHealthCheck', panel.idPanel());
-        system.assertEquals(null, panel.runDetector());
+        
+        System.assertEquals('idPanelHealthCheck', panel.idPanel());
+        System.assertEquals(null, panel.runDetector());
+
+        STG_Panel.stgService.stgErr.Disable_Record_Data_Health_Checks__c = true;
```

Comment:
Should saving settings be another test, wondering why it was added to an existing test? Or perhaps the `panel.runDetector()` should be executed after settings are saved?

Reply:
When writing the test, I notice there are a series of tests calles testPanel....(), and for all of them, it's testing the main functionality of each NPSP page. That's why I added this chunk of code into this part. It is technically a duplicate of `shouldCreateWarningRecordsWhenDisabledRecordHealthCheckIsEnabled`. Will remove it